fpdf/svg.py

```diff
@@ -920,6 +920,13 @@ def build_group(self, group, pdf_group=None):
                 self.handle_defs(child)
             elif child.tag in xmlns_lookup("svg", "g"):
                 pdf_group.add_item(self.build_group(child), False)
+            elif child.tag in xmlns_lookup("svg", "a"):
```

Comment:
I haven't checked the SVG spec, but if a `<a>` tag can also be present in `<defs>`, this code block should also be added to `SVGObject.handle_defs()`.

Reply:
I see the <a> tag explicitly mentioned in the list of children: https://www.w3.org/TR/SVG2/struct.html#DefsElement. I'll add it there too!